crates/symbolicator/src/services/objects/mod.rs

```diff
@@ -394,6 +395,8 @@ fn object_has_features(meta_handle: &ObjectMetaHandle, purpose: ObjectPurpose) -
             ObjectPurpose::Source => meta_handle.features.has_sources,
         }
     } else {
+        // TODO: shouldn't this be false if the cache status isn't positive? it has no features, not
+        // some features?
         true
```

Comment:
this isn't entirely related to the changes in this PR, but this seems a little strange. does another fetch for an object occur after `select_meta` chooses it? do we make an attempt to re-fetch a negative cache entry if it is determined to have the best quality out of all of the candidates?